docs/UserGuide.md

```diff
@@ -199,21 +196,19 @@ But wait! Notice the `[]` around `pr/PRIORITY`?
 | None    | Compulsory field | `C/COMPANY_NAME` | The company's name is required when adding a company  |
 | `[ ]`   | Optional field   | `[pr/PRIORITY]`  | Not a must to indicate priority when adding a company |
 
-Since `PRIORITY` parameter is optional, the following command lines are accepted.
+Since `PRIORITY` parameter is optional, the following command lines are accepted for the `add` command.
 
 **Example:**
 - `add c/GovTech r/Software Engineer s/PA n/John Doe d/12-12-2023 e/johnDoe@gmail.com p/98765432 pr/medium`
 - `add c/GovTech r/Software Engineer s/PA n/John Doe d/12-12-2023 e/johnDoe@gmail.com p/98765432`
 
 <div markdown="block" class="alert alert-info">
 **:information_source: Notes:**<br>
-* Parameters can be in any order. e.g. if the command specifies c/COMPANY_NAME r/ROLE, r/ROLE c/COMPANY_NAME is also acceptable.
+* Parameters can be in any order. e.g. If the command specifies c/COMPANY_NAME r/ROLE, r/ROLE c/COMPANY_NAME is also acceptable.
```

Comment:
Suggestion: add code blocks for `c/COMPANY_NAME r/ROLE`, and  `r/ROLE c/COMPANY_NAME`, for better clarity